deployments/container/Dockerfile.ubi9

```
@@ -43,11 +43,22 @@ ARG VERSION="N/A"
 ARG GIT_COMMIT="unknown"
 RUN make PREFIX=/artifacts cmds
 
-FROM nvcr.io/nvidia/cuda:12.8.0-base-ubi9
+# Download must-gather dependency: `kubectl`
+ARG TARGETARCH
+RUN OS_ARCH=${TARGETARCH/x86_64/amd64} && OS_ARCH=${OS_ARCH/aarch64/arm64} && \
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${OS_ARCH}/kubectl && \
+    chmod +x ./kubectl
 
-COPY --from=build /artifacts/k8s-kata-manager /usr/bin/k8s-kata-manager
+FROM nvcr.io/nvidia/distroless/go:v3.1.7-dev
```

Let's use the latest tag available.
```suggestion
FROM nvcr.io/nvidia/distroless/go:v3.1.8-dev
```